### PR TITLE
Added FluentUI Slider to chat settings panel for overriding temperature

### PR DESCRIPTION
**Pull request by bscheurm**
## Purpose

* The Chat Settings panel does not provide a way to override the temperature value for requests to the Azure OpenAI completion endpoint. This PR adds a slider to the panel for customizing the temperature. Note that no other changes are required other than UI because the existing code already supports a temperature override value, and the various approaches (located in app/backend/approaches/*) already use it.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by pamelafox** [comment]
Sorry for the very long delay! I'm going to review this shortly.

**Review by pamelafox — APPROVED** [review]
I've merged from main and added a slider for the Ask tab as well, with each default set to match the default in the Python files. We may change that in a future PR to be 0.3 across the board.

**Comment by pamelafox** [comment]
I've updated the temperature to 0.3 to match latest changes in main.

**Comment by bscheurm** [comment]
Thanks for all your work on this @pamelafox . Should I go ahead and complete this then?

**Review by pamelafox — APPROVED** [review]

**Comment by pamelafox** [comment]
I've approved the PR, just want @mattgotteiner to take a look to make sure he's on board.

File: app/frontend/src/pages/chat/Chat.tsx [hunk 1]
```diff
@@ -373,6 +383,18 @@ const Chat = () => {
                         onChange={onPromptTemplateChange}
                     />
 
+                    <Slider
+                        className={styles.chatSettingsSeparator}
+                        label="Temperature"
+                        min={0}
+                        max={1}
+                        step={0.1}
```
Comment (mattgotteiner) [inline app/frontend/src/pages/chat/Chat.tsx:391]: seems like a reasonable step to me

**Review by mattgotteiner — APPROVED** [review]

**Pull request merged**